Update docs for Spice.ai Catalog Connector (#723)
spiceai/docs: spiceaidocs/docs/components/catalogs/spiceai.md

pagination_next: null
---

> Query datasets provided by the [Spice.ai Cloud Platform](https://spice.ai).

Reviewer: ```suggestion
Query datasets hosted in the [Spice.ai Cloud Platform](https://spice.ai).
```

pagination_next: null
---

> Query datasets hosted in the [Spice.ai Cloud Platform](https://spice.ai). Discover available public datasets in the [Spicerack](https://spicerack.org).

Reviewer: ```suggestion
Query datasets hosted in the [Spice.ai Cloud Platform](https://spice.ai). Discover available public datasets on [Spicerack](https://spicerack.org).
```
